float32 current_actual_pressure
float32 current_set_pressure

bool compressor_on
bool ball_detected

bool motor_power_on
